src/views/PurchaseOrderDetail.vue

```diff
@@ -109,7 +109,9 @@
           <ion-text v-else color="medium" class="ion-margin-end">
             {{ $t("COMPLETED: ITEM", { itemsCount: getPOItems('completed').length }) }}
           </ion-text>
+          <ion-button fill="clear" >
```

Comment:
We have a click event on our ion-icon, if we wrap it in ion-button the event can't be triggered. Also UI changes a bit with ion-button.